docker/constraints-odc.txt

```
@@ -1,21 +1,10 @@
-# PyPi
 datacube==1.8.5
 odc-algo==0.2.0a3
-odc-apps-cloud==0.2.0a1
-odc-apps-dc-tools==0.2.0a1
+odc-cloud==0.2.0a0
 odc-dscache==0.2.0a1
-odc-geom==0.2.0a0
 odc-io==0.2.0a0
-odc_stats==0.2.0a2
+odc-stac==0.2.0a4
```

can we please bump this one:
```suggestion
odc-stac==0.2.0a5
```